build-tools/packages/build-cli/src/library/git.ts

```diff
@@ -3,15 +3,73 @@
  * Licensed under the MIT License.
  */
 
+import { execFileSync } from "node:child_process";
 import path from "node:path";
 import { Package } from "@fluidframework/build-tools";
+import { PackageName } from "@rushstack/node-core-library";
 import readPkgUp from "read-pkg-up";
+import * as semver from "semver";
 import { SimpleGit, SimpleGitOptions, simpleGit } from "simple-git";
 import type { SetRequired } from "type-fest";
 
+import { parseISO } from "date-fns";
 import { CommandLogger } from "../logging.js";
 import { ReleaseGroup } from "../releaseGroups.js";
-import { Context } from "./context.js";
+// eslint-disable-next-line import/no-deprecated
+import { Context, type VersionDetails, isMonoRepoKind } from "./context.js";
+
+const newlineCrossPlatform = /\r?\n/;
+/**
+ * Parses the version from a git tag.
+ *
+ * @param tag - The tag.
+ * @returns The version string, or undefined if one could not be found.
+ *
+ * TODO: Need to reconcile slightly different version in version-tools/src/schemes.ts
```

Comment:
Calling out in case we need to address this before merging.

Reply:
Not for this change, no. Thanks for checking!